test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -491,6 +505,10 @@ RandomCodeOptions::RandomCodeOptions() :
 	addAddress(Address("0x0000000000000000000000000000000000000006"), AddressType::Precompiled);
 	addAddress(Address("0x0000000000000000000000000000000000000007"), AddressType::Precompiled);
 	addAddress(Address("0x0000000000000000000000000000000000000008"), AddressType::Precompiled);
+	addAddress(Address("0x0000000000000000000000000000000000000005"), AddressType::ByzantiumPrecompiled);
```

Comment:
do we really need to distinguish precompiled from byzantium and other forks ?  
you could just initialize options with only byzantium precompiles

Reply:
We have to distinguish them to give Byzantium precompiles higher weights (sampling probability).

Reply:
you could define options with precompiled that are exist only in Byzantine. 
the goal is to make options configurable.